lib/entities/webhook.ts

```diff
@@ -28,33 +28,36 @@ export type WebhookProps = {
   /**
    * Username for basic http auth
    */
-  httpBasicUsername: string
+  httpBasicUsername: string | null
```

Comment:
I think this should be `httpBasicUsername?: string` because the property will omitted from the payload if it does not exist

Reply:
updated